pkg/tasks/tasks.go

```diff
@@ -90,6 +92,14 @@ func NewTaskGroup(tasks []*TaskSpec) *TaskGroup {
 	}
 }
 
+func (tge TaskGroupErrors) Error() string {
+	var messages []string
+	for _, err := range tge {
+		messages = append(messages, fmt.Sprintf("%v", err.Err))
```

Comment:
```suggestion
		messages = append(messages, fmt.Sprintf("%v: %v", err.Name, err.Err))
```

Reply:
Task Name is added when the error is sent through the channel.